package.json

```
@@ -156,6 +156,7 @@
     "@testing-library/react-hooks": "8.0.0",
     "@testing-library/user-event": "14.1.1",
     "@types/final-form-focus": "1.1.2",
+    "@types/node": "^17.0.31",
```

```suggestion
    "@types/node": "17.0.31",
```

18 would not be compatible ?